### PR TITLE
WIP: Add tests for alert "RHODS Probe Success Burn Rate"

### PR DESCRIPTION
**Pull request by jgarciao**
- Add keywords to start and stop the operator  
- Add tests for alert "RHODS Probe Success Burn Rate":
  - Adds tests ODS-712 and ODS-713
  - Creates keyword Wait Until Alert Is Firing
  - Refactors existing alerts tests to use it

### CODE REVIEW

File: tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  [Tags]  Sanity
+  ...    PLACEHOLDER  #Category tags
+  ...    ODS-517
+  Set Up PVC Alert Test  ${notebook_100}
+  Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}  RHODS-PVC-Usage  ${alert_100}
+  [Teardown]  End PVC Alert Test
+
+
+Verify alert "RHODS Probe Success Burn Rate" fires when rhods-dashboard service is down
+  [Tags]  LongRunning
```
Comment (vasukulkarni) [inline tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot:53]: we can use "Tier2" for this test

Comment (jgarciao) [reply tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot]: Modified

**Comment by aloganat** [comment]
Can one of the admins verify this patch?

**Comment by jgarciao** [comment]
These tests were finally added in another PR

**Pull request closed without merging**